recipes/liburing/all/conanfile.py

```diff
@@ -35,6 +37,8 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.version < "2.2":
```

Comment:
```suggestion
        if Version(self.version) < "2.2":
```
Need to compare the version, not string.